cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -187,6 +201,13 @@ message CassandraClusterState {
      * List of IP addresses of nodes that need to be replaced.
      */
     repeated string replaceNodeIps = 4;
+
+    /** Time of last fetching of seeds from ExternalDcs */
+    optional int64 lastSeedFetchTime = 5;
+
+    /** List of external seeds fetched from ExternalDc(s) via REST API */
+    repeated string externalSeeds = 6;
```

Comment:
Can we move these properties into the new `ExternalDc` message so that they can be tracked for each datacenter individually.

I'm imagining a scenario, where you are running cassandra in 3 DC and one of the DC becomes unavailable due to some network outage.  We should probably continue use the last known seed set for that DC even if we aren't able to re-query it at this point in time.

This could also be something we add to the health check report to surface whether this cluster can communicate with the other clusters and when the last communication was.